scripts/plot_KPIs.py

```diff
@@ -43,6 +41,35 @@
 rc('ytick', **{'labelsize': 12})
 # plt.style.use(["ggplot"])
 
+def test_run_latex():
```

Comment:
Maybe we want to name this function a bit more transparently since we are also possibly enabling latex mode in this function. 
```suggestion
def test_and_enable_latex():
```